panic when passing `.` for the path

```
C:\>gofind foo .
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x616f40, 0x750c50)
        c:/go/src/runtime/panic.go:527 +0x1ae
main.main()
        c:/dev/go/src/github.com/motemen/gofind/cmd/gofind/main.go:118 +0x1b56
```

--- a/cmd/gofind/main.go
+++ b/cmd/gofind/main.go
@@ -104,6 +104,10 @@ func main() {
 
 	paths := strings.Split(target, "/")              // {"golang.org","x","tools","go","loader.Config"}
 	names := strings.Split(paths[len(paths)-1], ".") // {"loader","Config"}
+	if len(names) < 2 {
+		log.Println("Query must be of form <pkg>.<name>[.<sel>]")
+		os.Exit(2)
+	}
 
 	// TODO(motemen): provide filename-only option like "grep -l"
 

